scripts/functions/detect_systems/os_release

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+__rvm_detect_system_from_os_release()
+{
+  local __system_name="$( awk -F'=' '$1=="ID"{print $2}' /etc/os-release | head -n 1 | tr '[A-Z]' '[a-z]' | tr -d \" )"
+
+  case $__system_name in
+    opensuse*)
+      _system_name="OpenSuSE"
+      _system_version="$( awk -F'=' '$1=="VERSION_ID"{gsub(/"/,"");print $2}' /etc/os-release | head -n 1 )"
```

Comment:
arch is taken from [`uname -m`](https://github.com/rvm/rvm/pull/3938/files#diff-6b0f39316be80fe9212e3309137eee62R17)